MantidQt/CustomInterfaces/src/Reflectometry/ReflSettingsTabPresenter.cpp

```diff
@@ -16,7 +16,7 @@ using namespace Mantid::Geometry;
 * @param view :: The view we are handling
 */
 ReflSettingsTabPresenter::ReflSettingsTabPresenter(IReflSettingsTabView *view)
-    : m_view(view), m_mainPresenter() {
+    : m_view(view), m_mainPresenter(), currentInstrumentName("INTER") {
```

Comment:
As discussed earlier, what will happen if my default instrument is for example `POLREF`? Will the `ReflSettingsTabPresenter` think that it is `INTER` and get defaults for `INTER` instead of `POLREF`?